packages/devtools_app_shared/lib/src/ui/buttons.dart

```diff
@@ -245,11 +245,10 @@ final class DevToolsToggleButton extends StatelessWidget {
           message: message,
           child: Container(
             padding: const EdgeInsets.symmetric(horizontal: denseSpacing),
-            child: MaterialIconLabel(
-              iconData: icon,
-              label: label,
-              minScreenWidthForTextBeforeScaling:
-                  minScreenWidthForTextBeforeScaling,
+            child: ImageIconLabel(
```

Comment:
is this the correct fix here? or is the fix to apply the color properly in material icon label? `devtools_app_shared` is a public package so we should be careful about making changes here that may potentially break downstream UIs

Reply:
Updated `MaterialIconLabel` instead. This will however change the appearance of the label for any clients who didn't specify a `color`. Previously if `color` was `null`, the Flutter framework would decide on the color for the icon based on the `iconTheme`, but the text label color wouldn't match because the `color` there is `null`. 

I think from the code it was intended that the icon and its label would always have the same color. If we want to only fix the `DevtoolsToggleButton` case I could explicitly pass in the `color` parameter.

Reply:
Can we fix this without making changes to devtools_app_shared?

Reply:
Done - I added some optional parameters, but this shouldn't affect any existing widgets from devtools_app_shared